client/systray/systray.go

```diff
@@ -182,6 +192,15 @@ func (menu *Menu) rebuild() {
 
 	systray.ResetMenu()
 
+	if menu.readonly {
+		const readonlyMsg = "No permission to manage Tailscale.\nSee tailscale.com/s/cli-operator"
+		m := systray.AddMenuItem(readonlyMsg, "")
+		onClick(ctx, m, func(_ context.Context) {
+			webbrowser.Open("https://tailscale.com/s/cli-operator")
+		})
+		systray.AddSeparator()
+	}
```

Comment:
would it be simpler to just add the Quit button here and return early?
with those individual `menu.readonly` checks below it will be easy to add a new menu item later and forget to disable it.

Reply:
yeah, we certainly could.  I suppose it depends on what experience we want here.  For example, quite a few CLI commands still work fine without operator set such as `tailscale status` and even `tailscale exit-node list`.  I was trying to do something similar hear and leave the menu items that technically still work.

If we did forget to gate some new menu item in the future, it would likely just be non-functional at worst.  But in practice, I don't think it's going to be very common that someone would actually want to run the systray app without operator set, so maybe it's not worth the effort here.  🤷🏻

Reply:
Ah, right, i forgot that only some stuff requires an operator status.
Nvm, this is good as is :+1: